Anchor.toml

```diff
@@ -5,13 +5,13 @@ resolution = true
 skip-lint = false
 
 [programs.localnet]
-multicall_handler = "27dua7HMuaZrAc6PRfxmvshHChypFLzZVSpKKaDAPjsq"
-svm_spoke = "D1WcTLKRyN4TuqnfMBGPvc16nACHfNhFPbgWNExb1aur"
+multicall_handler = "Fk1RpqsfeWt8KnFCTW9NQVdVxYvxuqjGn6iPB9wrmM8h"
+svm_spoke = "JAZWcGrpSWNPTBj8QtJ9UyQqhJCDhG9GJkDeMf5NQBiq"
 test = "8tsEfDSiE4WUMf97oyyyasLAvWwjeRZb2GByh4w7HckA"
 
 [programs.devnet]
```

Comment:
should there be a section in here for main net with the program address now that its on mainnet?